Exercise_B/index.html

```diff
@@ -1,34 +1,71 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <title>Flower Shop BS</title>
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <!--If you want to use bootstrap properly this is vital (helps browser to measure the page and readjust the information accordingly-->
-
-    <!-- Link and CDN required for bootstrap -->
-
-    <!-- Google Font - Lato -->
-    <link
-      href="https://fonts.googleapis.com/css?family=Lato"
-      rel="stylesheet"
-    />
-    <!-- Custom Stylesheet -->
-    <link rel="stylesheet" href="./assets/style/stylesheet.css" />
-  </head>
-
-  <body>
-    <!--Header (flower shop logo block element)-->
-    <!--Banner Jumbotron of the image-->
-
-    <!--Four containers with img / text-->
-    <!-- 4 containers -->
-
-    <!-- Payments Div -->
-    <!-- Consultation Div -->
-
-    <!-- Delivery Div -->
-
-    <!--Scripts required for bootstrap CDN-->
-  </body>
-</html>
+
+<head>
+  <meta charset="utf-8" />
+  <title>Flower Shop BS</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <!--If you want to use bootstrap properly this is vital (helps browser to measure the page and readjust the information accordingly-->
+
+  <!-- Link and CDN required for bootstrap -->
+
+  <!-- Google Font - Lato -->
+  <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet" />
+  <!-- Custom Stylesheet -->
+  <link rel="stylesheet" href="./assets/style/stylesheet.css" />
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-+0n0xVW2eSR5OomGNYDnhzAbDsOXxcvSN1TPprVMTNDbiYZCxYbOOl7+AMvyTG2x" crossorigin="anonymous">
+</head>
+
+<body>
+  <section class="header">
+    <nav>
+      <a href="index.html"><img src="./assets/images/logo.png"></a>
+    </nav>
+    <div class="text-box">
+      <h1>Beautiful Flowers</h1>
+      <a href="http://www.9gag.com" target="_blank" class="learnMoreBtn">Learn More</a>
+    </div>
+
+  </section>
+
+  <section class="info">
+    <div class="container">
+      <div class="row">
+        <div class="col-12  col-md-6 col-lg-3">

```

Comment:
I would alter this class a little so you can have the icon and text on the same line, consider using flex or altering the p tags to inline elements.